client/components/forms/counted-textarea/index.jsx

```diff
@@ -70,7 +70,14 @@ export default React.createClass( {
 		return (
 			<div className={ classes }>
 				<FormTextarea
-					{ ...omit( this.props, 'className', 'acceptableLength', 'showRemainingCharacters', 'children' ) }
+					{ ...omit(
+						this.props,
+						'className',
+						'acceptableLength',
+						'showRemainingCharacters',
+						'children',
```

Comment:
It was there before, but we don't need to explicitly omit `className` or `children` because we're overriding them in the next few lines anyways. Not a blocker.